### ISSUE TITLE
composer.json invalid

### ISSUE THREAD

**Issue by bryjbrown** [kent-state-university-libraries/rest_oai_pmh#1]
Hello! I'm interested in your project and decided to give it a whirl on a VM this morning and found that Composer doesn't like your composer.json file. Parsing the file throws errors at [line 15](https://github.com/kent-state-university-libraries/rest_oai_pmh/blob/8.x-1.x/composer.json#L13-L15), because you are using curly brackets to denote an associative array, but inside you have regular array values (eg, not "key" : "value" pairs).

I'm no composer expert, but comparing with [the 'require' section of Islandora 8's composer.json file](https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/composer.json#L17-L33), it looks like you are missing a stated version for `drupal/metatag'. Adding some sort of basic version requirement, such as `"drupal/metatag" : "^1.8"`, would make the composer.json valid.

I would have made this a PR, but I'm not sure if this module actually does require a specific version of metatag or not, so I leave it up to you.

**Comment by joecorall** [comment]
Hey @bryjbrown 

Thanks for your interest in the module! It's still in very early stages of development - though I hope to have an alpha release by this Wednesday along with documentation on how to install // configure.

Though I did push a fix to the composer.json this morning. Reading closer into Drupal's documentation, I don't think they require specifying module dependencies in composer.json if the dependency is in *.info.yml, so I removed the "require" statement from composer.json altogether.
